plugins/inputs/mqtt_consumer/README.md

```diff
@@ -194,5 +194,15 @@ sensors,site=CLE,version=v1,device_name=device5 temp=390,rpm=45.0,ph=1.45
 
 - example when [[inputs.mqtt_consumer.topic_parsing]] is set
 
+- when [[inputs.internal]] is set:
+  - bytes_received (int): count of the number of bytes have been received from incoming messages
+  - messages_received (int): count of the number of messages that have been received from mqtt
+  
+This will result in the following metric:
+
+```text
+internal_mqtt_consumer,host=pop-os,version=1.24.0-2a266e53,messages_received=622i,bytes_received=37942i 1657282270000000000
```

Comment:
```suggestion
internal_mqtt_consumer,host=pop-os,version=1.24.0-2a266e53 messages_received=622i,bytes_received=37942i 1657282270000000000
```
There needs to be a space to delimit the tags and fields, I would also remove the hash from the version